core/src/main/java/org/apache/carbondata/core/datastore/impl/FileFactory.java

```diff
@@ -111,14 +112,9 @@ public static DataInputStream getDataInputStream(String path, FileType fileType)
     return getDataInputStream(path, fileType, -1);
   }
 
-  public static DataInputStream getDataInputStream(String path, FileType fileType,
```

Comment:
Why do you remove this interface?

Reply:
Oh thats a rebase mistake..I think need to rebase again.

Reply:
ok, please rebase asap